File: introduction/ITP1_3_B-Print Test Cases.cpp
#include <iostream>
using namespace std;

int main(void) {
  int count = 1, line;
  cin >> line;
  while (line != 0) {
    cout << "Case " << count << ": " << line << endl;
    cin >> line;
    ++count;
  }
  return 0;
}

File: introduction/ITP1_3_B-Print Test Cases.scala
object Main {
  def main(args: Array[String]):Unit = {
    var count = 1
    for(line <- scala.io.Source.stdin.getLines() if line.toInt != 0) {
      println("Case " + count + ": " + line)
      count += 1
    }
  }
}
